src/ui/components/checklists/Checklists.js

```diff
@@ -525,6 +544,15 @@ class Checklists extends Component {
                             <BlockUi blocking={blocking}>
                                 {this.props.topSlot}
                                 <div style={{paddingLeft: 25, paddingRight: 25}}>
```

Comment:
```suggestion
<BlockUi blocking={blocking}>
                                {this.props.topSlot}
                                <div style={{paddingLeft: 25, paddingRight: 25}}>
                                {<FormControlLabel
                                        control={<Checkbox
                                            color="primary"
                                            checked={Object.keys(this.state.checklistsHidden).length > 0}
                                            onMouseDown={this.toggleFilledFilter}
                                            onTouchStart={this.toggleFilledFilter}
                                            />}
                                        label={'Hide filled items'}
                                    />}
```
I would have the the topSlot and the checkbox on a single row, wrapping topSlot in a div with width 50% is worth exploring.